src/features/Favourites/components/Favourites/Favourites.test.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import {fireEvent, render} from '@testing-library/react-native';
+import Favourites from './Favourties';
+import * as redux from 'react-redux';
+import {IJoke} from '../../../../models/IJoke';
+import {changeSortingType, SORTING_TYPE} from '../../../../actions/sortActions';
+import {removeFromFavourites} from '../../../../actions/favouritesActions';
+jest.useFakeTimers();
+jest.mock('react-native-vector-icons/FontAwesome', () => 'icon');
+jest.mock('react-native-vector-icons/Ionicons', () => 'icon');
+jest.mock('react-native-vector-icons/MaterialCommunityIcons', () => 'icon');
+
+jest.mock('react-redux', () => ({
```

Comment:
Seems like a bad practice.There is a simpler way to mock useDispatch.
const useDispatchSpy = jest.spyOn(redux, 'useDispatch');
useDispatch.mockImplementation(jest.fn())

Reply:
I guess this is a leftover I forgot to delete since I did jest.spyon usedispatch in this file.
deleted :)

Reply:
Seems like you still have to mock the library itself since it fails on: cannot redefine property:  useSelector

https://github.com/aelbore/esbuild-jest/issues/26